exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+var numberOfStudents = 15 / 23;
```

Comment:
Just a tip to keep the original request before you change it. Its not wrong but just helps for troubleshooting. So for e.g. perhaps you could have had:
var numberOfStudents = 15 ;
numberOfStudents = 15/23

Also need to multiply by 100 to get percentage before using Math.round. So perhaps in addition
studentsPercentage = Math.round(numberOfStudents * 100)
console.log( `{studentsPercentage}%` )